### PR TITLE
Update Mesos doc

### PR DESCRIPTION
**Pull request by tieujason330**

### CODE REVIEW

**Comment by tieujason330** [comment]
jenkins, test this please

**Comment by tieujason330** [comment]
@cheng-chang, PTAL thanks

File: docs/en/deploy/Running-Alluxio-On-Mesos.md [hunk 1]
```diff
@@ -74,12 +74,18 @@ From anywhere with Alluxio installed:
 
 {% include Running-Alluxio-on-Mesos/alluxio-mesos.md %}
 
-Note that the tarball should be compiled with `-Pmesos`. Released Alluxio tarballs from version 1.3.0 onwards are compiled this way.
+- Note that the tarball should be compiled with `-Pmesos`. Released Alluxio tarballs from version 1.3.0 onwards are compiled this way.
+- Mesos library path is assumed to be at `/usr/lib`. If it is not, you can update `MESOS_LIBRARY_PATH` in `alluxio-mesos-start.sh` with the correct path
```
Comment (cheng-chang) [inline docs/en/deploy/Running-Alluxio-On-Mesos.md]: end with `.`

File: docs/en/deploy/Running-Alluxio-On-Mesos.md [hunk 2]
```diff
@@ -74,12 +74,18 @@ From anywhere with Alluxio installed:
...
+
+Verify that the Alluxio Mesos framework is working by starting a Mesos master and confirming that it is detected in `alluxio/logs/framework.out`
```
Comment (cheng-chang) [inline docs/en/deploy/Running-Alluxio-On-Mesos.md]: end with `.`

**Comment by cheng-chang** [comment]
alluxio-bot, merge this please

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**